### PR TITLE
feat/new markets may 15

### PR DESCRIPTION
**Pull request by carlomigueldy**
- feat: add new market assets
- feat: add new market objects
- feat: add support for multi chain on new type defs in Market
- feat: attach testnet data

### CODE REVIEW

**Comment by valentynkvasov** [comment]
good job man!

**Comment by oleksandrtarapata** [comment]
please pay attention after market type changed everywhere we display market.tokenTicket and market.tokenURI should be new type

File: src/constants/markets.ts [hunk 1]
```diff
@@ -1,9 +1,303 @@
...
+      link: 'https://twitter.com/ZinedineYazidZi',
+    },
+    tags: ['Football', 'ChampionsLeague', 'Final', 'RealMadrid', 'Borussia'],
+  },
+
+  // correct
+  {
+    address: {
+      [base.id]: '0x96ace74aa1ae07f7baf4e7f2736cbe2a703712fc', // prod
+      [baseSepolia.id]: '0x', // testnet
```
Comment (oleksandrtarapata) [inline src/constants/markets.ts:183]: this errors on market page navigate

File: src/constants/markets.ts [hunk 2]
```diff
@@ -1,9 +1,303 @@
...
+      link: 'https://twitter.com/AutismCapital',
+    },
+    tags: ['LexFriedman', 'ayahuasca', 'AutismCapital'],
+  },
+
+  // correct
+  {
+    address: {
+      [base.id]: '0x5f3293417b0228bdc1896e34de99b377d31c77d8', // prod
+      [baseSepolia.id]: '0x',
```
Comment (oleksandrtarapata) [inline src/constants/markets.ts:222]: this too

**Comment by carlomigueldy** [comment]
> please pay attention after market type changed everywhere we display market.tokenTicket and market.tokenURI should be new type

Yes, no worries I already fixed those issues. The pre lint commit check have caught it and I already resolved it.

File: src/constants/markets.ts (hunk 2 above)
Comment (carlomigueldy) [reply src/constants/markets.ts]: Resolved it

File: src/constants/markets.ts (hunk 1 above)
Comment (carlomigueldy) [reply src/constants/markets.ts]: Resolved it

**Review by oleksandrtarapata — APPROVED** [review]

**Pull request merged**